src/test/js/karaoke-noStages.js

```diff
@@ -26,15 +26,19 @@ module.exports = {
         browser.waitForElementVisible('code')
             .click('code')
             .keys(browser.Keys.UP_ARROW)
-            .getText('code', function (result) {
-                const text = result.value;
-                this.pause(1000)
-                    .waitForElementVisible('code')
-                    .getText('code', function (result) {
-                        this.assert.equal(text, result.value)
-                    });
-
-            });
+            .elements('css selector', 'div.result-item', function (resutlItems) {
+                var results = resutlItems.value.length;
+                // to validate that we left follow, give it some time and then count the elements again
+                this.pause(3000)
+                    .elements('css selector', 'code', function (codeCollection) {
+                        // JENKINS-36700 there can only be one code view open in follow stopped
+                        this.assert.equal(codeCollection.value.length, 1);
+                    })
+                    .elements('css selector', 'div.result-item', function (resutlItemsCompare) {
+                        // there should not be more items then we had before
+                        this.assert.equal( resutlItemsCompare.value.length, results);
+                    })
+                });
```

Comment:
Yeah, I do not like the pause either but to test whether we are not following anymore, I do not see any other way. The 3sec it to wait long enough to make sure we really stopped. The whole idea of stop karaoke is that there will not be any new Element I could wait on other then the end of the pipeline. ATM I  do not see the re usability